### PR TITLE
PM-1712 - Fix mina-archive values

### PR DESCRIPTION
**Pull request by johnmarcou**

### CODE REVIEW

File: mina-archive/values.yaml [hunk 1]
```diff
@@ -140,7 +140,7 @@ missingBlocksGuardian:
 
 dumpExporter:
   # -- Enabled dump exporter 
-  enabled: true
+  enabled: false
   # -- Suspend the  dump exporter execution
   suspend: false
```
Comment (johnmarcou) [inline mina-archive/values.yaml:145]: We could also enabled both Jobs by default, but suspend them. That could be handy if we decide to run a job manually for any reason.

Comment (kaozenn) [reply mina-archive/values.yaml:145]: I agree with it.

**Review by kaozenn — APPROVED** [review]
**praise**: thanks for updating the chart to match the post HF state

**Review by Smorci — APPROVED** [review]
LGTM

File: mina-archive/scripts/wait-for-db.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ main() {
   info "Locking database: $DB_BOOTSTRAP_LOCKING_DATABASE_NAME"
 
   info "Checking if database server is online"
-  retry_while check_database_is_online 60 1 # 1 min
```
[the remark this answers is not in the document]
Comment (johnmarcou) [reply mina-archive/scripts/wait-for-db.sh:20]: Good catch! Thank you. Applied.

File: mina-archive/values.yaml (hunk 1 above)
Comment (johnmarcou) [reply mina-archive/values.yaml:145]: After discussing it, we agree that:
- guardian should be `enabled` by default, and not suspended
- dump exporter should **NOT** be `enabled`, and not suspended

**Pull request merged**